anesthetic/plotting/_matplotlib/__init__.py

```diff
@@ -0,0 +1,43 @@
+"""anesthetic override of matplotlib backend for weighted samples."""
```

Comment:
This imports all of the things it needs to from the matplotlib backend, and replaces the relevant functions and classes with functions defined in anesthetic._matplotlib

Reply:
These are indeed included, since we import the `PLOT_CLASSES` from pandas, and then overwrite the code which needs modification. Since I don't _think_ the other classes (namely `line`, `bar`, `barh`, `area` and `pie`).

```python
>>> from anesthetic.plotting._matplotlib import PLOT_CLASSES
>>> print(PLOT_CLASSES)
{'line': pandas.plotting._matplotlib.core.LinePlot,
 'bar': pandas.plotting._matplotlib.core.BarPlot,
 'barh': pandas.plotting._matplotlib.core.BarhPlot,
 'box': anesthetic.plotting._matplotlib.boxplot.BoxPlot,
 'hist': anesthetic.plotting._matplotlib.hist.HistPlot,
 'kde': anesthetic.plotting._matplotlib.hist.KdePlot,
 'area': pandas.plotting._matplotlib.core.AreaPlot,
 'pie': pandas.plotting._matplotlib.core.PiePlot,
 'scatter': anesthetic.plotting._matplotlib.core.ScatterPlot,
 'hexbin': anesthetic.plotting._matplotlib.core.HexBinPlot}
```

Giving it a bit more thought, there may be some instances of e.g. pie when computing down a column you might want weighted versions of these -- will get back to you here.